MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1450,10 +1432,10 @@ public void writeToXml(PrintWriter pw1, int indent) {
                         + df.format(recruitment.getTime())
                         + "</recruitment>");
         }
-        for (String skName : skills.keySet()) {
+        skills.keySet().stream().sorted().forEach(skName -> { // sorted to get consistent .cpnx files
```

Comment:
Another "opportunity" change :-)

I was diffing savefiles to check the force icon stuff and the changes in order of skills kept popping up as false positives